specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintAssignment.json

```diff
@@ -424,6 +424,7 @@
       "type": "object",
       "properties": {
         "value": {
+          "type": "string",
```

Comment:
@majastrz currently this is an object type, making it string. Would it work for the current use of it?

Reply:
We can't unfortunately. We allow any valid JSON value here including objects, arrays, strings, numbers and booleans.

---
In reply to: [379550473](https://github.com/Azure/azure-rest-api-specs/pull/8404#discussion_r379550473) [](ancestors = 379550473)

Reply:
> We can't unfortunately. We allow any valid JSON value here including objects, arrays, strings, numbers and booleans.
> 
> In reply to: [379550473](https://github.com/Azure/azure-rest-api-specs/pull/8404#discussion_r379550473) [](ancestors = 379550473)

Thanks. Will update accordingly.